discovery-infra/tests/test_kube_api.py

```diff
@@ -0,0 +1,292 @@
+import base64
+import errno
+import json
+import logging
+import os
+import socket
+import uuid
+from copy import deepcopy
+from ipaddress import IPv4Interface, IPv6Interface
+
+import magic
+import openshift as oc
+import pytest
+import waiting
+from netaddr import IPNetwork
+from test_infra import consts
+from test_infra.helper_classes.kube_helpers import (
+    InstallStrategy, Proxy, deploy_default_cluster_deployment,
+    deploy_default_infraenv)
+from test_infra.utils import download_iso
+
+from tests.base_test import BaseTest
+from tests.conftest import env_variables
+
+PROXY_PORT = 3129
+
+logger = logging.getLogger(__name__)
+
+
+class TestKubeAPISNO(BaseTest):
+
+    @pytest.mark.kube_api
+    def test_kube_api_sno_happy_flow_create_and_install_cluster(
```

Comment:
I would remove the part of ``_create_and_install_cluster`` if we already have ``happy_flow``

Reply:
also sno, as we currently don't have anything else